PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,11 @@
+## Checklist 🛎️
+
+- [ ] Add product, design, and eng for rdev review
+
+- [ ] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see
+
+- [ ] For UI changes, add e2e tests to prevent regressions
+
```

Comment:
Do we want this to appear _before_ the issue link? Would prefer issue at top.

Reply:
Ooh good point - moved to the bottom!